service/crawlingKeywordService.js

```diff
@@ -79,4 +87,40 @@ const getCrawlingKeyword = async (req, res) => {
   }
 };
 
+const isCheckTrueThisUrl = (url) => {
+  /* eslint-disable */
+  const urlRegex = /^(http|https):\/\/[\w\-]+(\.[\w\-]+)+[/#?]?.*$/;
+  if (urlRegex.test(url)) {
+    return true;
+  } else {
+    return false;
+  }
+};
+
+const getAllSentence = (innerText) => {
+  return innerText
+    .replace(/\n|\r|\t/g, " ")
+    .split(/(?<=다\. |요\. |니다\. |\. |! |\? )/)
+    .map((sentence) => sentence.trim())
+    .filter((sentence) => sentence);
+};
+
+const getKeywordSentence = (sentence, keyword) => {
+  const words = sentence.split(/\s+/);
+  const keywordIndex = words.findIndex((word) => word.includes(keyword));
+  const startWordIndex = Math.max(0, keywordIndex - 3);
+  const endWordIndex = Math.min(words.length, keywordIndex + 4);
```

Comment:
해당 키워드가 포함되어 있는 index를 기준으로 문장의 시작과 끝을 정하는 부분 같은데 이 부분의 3과 4는 무엇을 뜻하는지 궁금합니다.
또한 해당 부분의 3과 4가 바뀔 가능성이 있다면 변수로 관리하면 좋을 것 같은데 어떻게 생각하시는지 의견을 듣고 싶습니다.

Reply:
```suggestion
  const startWordIndex = Math.max(0, keywordIndex - theNumberOfWordBefore);
  const endWordIndex = Math.min(words.length, keywordIndex + theNumberOfWordAfter + 1);
```
3과 4는 키워드가 포함된 글을 보여 줄 때, 키워드 앞과 뒤에 몇 개의 단어를 보이게 할 지 정하기 위한 값입니다. 해당 코드에는 키워드 앞뒤로 3개의 단어를 보이게 하기 위해 값을 넣은 상태입니다. endWordIndex 값에 4가 사용된 이유는 해당 코드 이후 slice함수의 두 번째 파라미터로 사용되기 때문에 추가로 1을 더하였습니다. 추후에 변경될 수 있는 값이기 때문에 변수로 관리하고자 하시는 의견은 좋다고 생각합니다. 수정하여 반영하겠습니다. 감사합니다!